packages/ui5-proxy-middleware/src/base/proxy.ts

```diff
@@ -0,0 +1,53 @@
+import { createProxyMiddleware, Filter, Options } from 'http-proxy-middleware';
+import { ClientRequest, IncomingMessage, ServerResponse } from 'http';
+import { UI5Config } from './types';
+import { proxyRequestHandler, proxyResponseHandler } from './utils';
+import { ToolsLogger } from '@sap-ux/logger';
+import { UI5ToolingTransport } from '@sap-ux/logger/dist/transports';
```

Comment:
Is `UI5ToolingTransport` not exposed by the main file of the module? If not, could we please do that and change the from value to `@sap-ux/logger` so that we do not rely on the internal folder structure.

Reply:
It is exposed, I will fix the import.

Reply:
Done